whois-commons/src/main/java/net/ripe/db/whois/common/rpsl/DummifierNrtm.java

```diff
@@ -9,12 +9,12 @@
 import org.slf4j.LoggerFactory;
 import org.springframework.stereotype.Component;
 
-import java.util.Iterator;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 
-@Component
+@Component("dummifierNrtm")
 public class DummifierNrtm implements Dummifier {
```

Comment:
Do we need this change?

Reply:
yes, because there's more than one implementation of Dummifier